packages/office-addin-debugging/src/start.ts

```diff
@@ -219,6 +219,12 @@ export interface StartDebuggingOptions {
      */
     enableLiveReload?: boolean,
 
+    /**
```

Comment:
Will this be a breaking change given that we are introducing a new param?

Reply:
No, because we moved it to the options object, so the function params aren't changing.   😄